tests/github_actions/test_schema_validation.py

```diff
@@ -19,3 +19,14 @@ def test_schema_validation(bad_schema_files):
 
     assert all(results[filename] is True for filename in filenames - bad_schema_files)
     assert all(results[filename] is False for filename in bad_schema_files)
+
+
+def test_off_value_parsed_correctly():
+    # given
+    test_file = RESOURCE_DIR / "off_value.yaml"
+
+    # when
+    definition, _ = Runner()._parse_file(str(test_file))
+
+    # then
+    assert definition["jobs"]["pre-commit"]["steps"][1]["with"]["comment_mode"] is False
```

Comment:
This test asserts the incorrect behavior. In GitHub Actions, `"off"` is simply a string.

Reply:
yeah, I know that from our brief discussion in the original issue. Sadly I need to dig deeper to make it behave differently for GHA. Currently all the workflow files are parsed identically, so `off` becomes `False`

Reply:
Makes sense; that was certainly the right decision. If you attempt higher fidelity parsing of GitHub Actions, I recommend consulting with the [nektos/act](https://github.com/nektos/act) and [rhysd/actionlint](https://github.com/rhysd/actionlint) communities so Checkov doesn’t have to reinvent the wheel.